packages/fuels-abigen-macro/tests/test_projects/sway_native_types/Forc.toml

```
@@ -0,0 +1,7 @@
+[project]
+name = "sway_native_types"
+authors = ["Fuel Labs"]
```

This seems to be the standard in all other tomls
```suggestion
authors = ["Fuel Labs <contact@fuel.sh>"]
```